working-groups/learning/README.md

```diff
@@ -0,0 +1,36 @@
+# Learning Working Group
+![working group status: active][status]
+
+This working group aims to make compiler easier to learn by ensuring that rustc-guide and api docs are "complete".
+
+- **Leads:** [@nikomatsakis][nikomatsakis] and [@spastorino][spastorino]
+- **[Ideas][ideas]**
+- **Meeting Notes:** [All](NOTES.md)
+
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+[nikomatsakis]: https://github.com/nikomatsakis
+[spastorino]: https://github.com/spastorino
+[ideas]: https://paper.dropbox.com/doc/Learning-Working-Group-Ideas--Ab18Mvvg7fYl8b~WL_klryqnAg-2R49k1GWZFojHIQ5x1fuB
+
+## What is the goal of this working group?
+This working group aims to accomplish the following:
+
+- Ensure that major components of rustc are covered in rustc-guide
+- Ensure that API doc coverage is at least 90%
+
+# How can I get involved?
+If you are interested in getting involved in this working group, you should try attend a meeting and
+introduce yourself or send a message in the Zulip stream. You can be added to the GitHub and Zulip
+group for the working group if you are interested in being pinged when there are available tasks.
+
+## Meetings
+We follow a "sprint-like" structure where we on each "sprint" cycle we start with a planning and
+then work towards the goals defined on that plan.
+
+- **Desired experience level:** Any
+- **Relevant repositories:** [`rust-lang/rust`][rust-repo] and [rust-lang/rustc-guide][rustc-guide-repo]
+- **Zulip stream:** [`#t-compiler/wg-learning`][zulip] on Zulip
+
+[rust-repo]: https://github.com/rust-lang/rust
+[rustc-guide-repo]: https://github.com/rust-lang/rustc-guide
+[zulip]: https://rust-lang.zulipchat.com/#narrow/stream/<wg-learning-stream>
```

Comment:
Same, we need to open a stream, I don't have rights to do so.

Reply:
https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-learning